packages/core/src/audio/AudioSource.ts

```
@@ -242,4 +251,4 @@
     const isValidClip = this._clip?._getAudioSource() ? true : false;
     return isValidClip && AudioManager.isAudioContextRunning();
   }
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)